src/Examine.AzureDirectory/AzureDirectory.cs

```diff
@@ -85,13 +87,31 @@ public void ClearCache()
                 CacheDirectory.DeleteFile(file);
             }
         }
+        public void RebuildCache()
+        {
+            try
+            {
+
+                ClearCache();
+            }
+            catch (Exception e)
+            {
```

Comment:
Should we log something, or just not care?

Reply:
that is not really something what is problematic, but we can log that but it doesn't really change anything